tests/integration/nm/route_rule_test.py

```diff
@@ -192,13 +185,3 @@ def _check_ip_rules_exist_in_os(rules):
             rule.get(RouteRule.PRIORITY),
             rule.get(RouteRule.ROUTE_TABLE),
         )
-
-
-def _assert_route_rules(nm_plugin, expected_rules):
```

Comment:
Instead of delete it, please use `libnmstate.show()` for assertion.

Reply:
This test does not exist anymore.